register.go

```diff
@@ -23,7 +23,7 @@ func (s *Snitch) register(looper director.Looper) error {
 		ClientInfo: &protos.ClientInfo{
 			ClientType:     protos.ClientType(s.config.ClientType),
 			LibraryName:    "snitch-go-client",
-			LibraryVersion: "0.0.40",
+			LibraryVersion: "0.0.40", // TODO: Have CI bump this on release
```

Comment:
Having to bump this manually sucks.

Reply:
you don't have to bump, CI does this already: https://github.com/streamdal/snitch-go-client/blob/main/.github/workflows/main-test.yml#L27-L45

Reply:
Oh shit. Thank you! That is excellent!